src/components/Wizard/WizardStep.js

```diff
@@ -34,8 +34,8 @@ WizardStep.propTypes = {
   /** @type {function(state: Object): Promise} this is called after each step. You can do async work here or save your changes
    * depending on whether you are using a sequential wizard or not. */
   next: PropTypes.func,
-  /** @type {function(state: Object): boolean} Validator. When it returns true, the "Next" or "Save" button becomes active
-   * depending on whether you are using a sequential or not. */
+
+  /** @type {function(state: Object): boolean} Pure function for the contents of the wizard step that determines the overall wizard's state. When it returns `true`, parts of the wizard (buttons, navigation, etc.) become active. */
```

Comment:
It is important to mention, that this function runs with every state change of the wizard (ie. after every keystroke).
```suggestion
  /** @type {function(state: Object): boolean} Predicate to validate the wizards state of the current step. This function must be pure and synchronous. Validation happens after every state change (i.e. the passed function is being called after every keystroke of a user). Don't use this as a hook to trigger any side-effects, nor allow the return value to depend on anything else than the argument. When it returns `true`, parts of the wizard (buttons, navigation, etc.) become active. */
```